modules/local/fusioncatcher/detect/main.nf

```diff
@@ -2,8 +2,7 @@ process FUSIONCATCHER {
     tag "$meta.id"
     label 'process_high'
 
-    conda "${moduleDir}/environment.yml"
-    container "community.wave.seqera.io/library/fusioncatcher:1.33--4733482b637ef92f"
+    container "docker.io/rannickscilifelab/fusioncatcher:1.34"
```

Comment:
This is the container that should be used, the conda container is buggy and not updated

Reply:
I think that the cleanest approach would be to update the conda recipy and create a new wave container with the updated version. docker.io containers from private users are not typically used for modules, I think this is due to the small pull rate limit